Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -0,0 +1,854 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import warnings
+import numpy as np
+import re
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.base import BaseEstimator, TransformerMixin
+import json
+import pandas as pd
+from scipy.spatial.distance import cdist
+
+warnings.simplefilter("ignore")
+
+FIRST_COLUMNS_INCIDENTS_DISPLAY = ['ID', 'created', 'name', 'final_score', 'similarity indicators']
```

Comment:
Would add columns renaming to camel case (check their appearance on the incidents table)